src/main/resources/fileTemplates/j2ee/sponge/Sponge build.gradle.ft

```diff
@@ -5,8 +5,9 @@ plugins {
 group = '${GROUP_ID}'
 version = '${PLUGIN_VERSION}'
 
-sourceCompatibility = '1.8'
-targetCompatibility = '1.8'
+java {
+    toolchain.languageVersion.set(JavaLanguageVersion.of(8))
```

Comment:
This applies to each of the changes in the toolchains commit - Just making sure that there is no way a user could configure a project setup which would lead to one of these templates to be applied to a version of Gradle < 6.7?

Reply:
I have to test that, it might be an issue with FG2 projects.